src/testhost.x86/Program.cs

```diff
@@ -52,8 +52,9 @@ public static void Main(string[] args)
         // In UWP(App models) Run will act as entry point from Application end, so making this method public
         public static void Run(string[] args)
         {
-            WaitForDebuggerIfEnabled();
-            SetCultureSpecifiedByUser();
+            DebuggerBreakpoint.WaitForNativeDebugger("VSTEST_HOST_NATIVE_DEBUG");
```

Comment:
Some comment about possibly moving the names for these variables to sharedproj too..

Reply:
Why would you move those variables to shared project when they are not shared?

Reply:
Just to have them in one place.

Reply:
I don't agree with that. This would publish those constants to other projects that don't care about them. And those constants are used literally once in the whole code base. I don't want to have another ".common" library.